src/app/dynamic/enterprise/quotas/project-quota-dialog/component.ts

```diff
@@ -149,16 +178,31 @@ export class ProjectQuotaDialogComponent implements OnInit, OnDestroy {
         validators: Validators.required,
         nonNullable: true,
       }),
-      subjectName: this._builder.control(
-        {value: subjectName, disabled: true},
-        {validators: Validators.required, nonNullable: true}
-      ),
+      subjectName: this._builder.control(subjectName ?? '', {
+        validators: Validators.required,
+        asyncValidators: AsyncValidators.ProjectSelection,
+        nonNullable: true,
+      }),
+    });
+
+    this.projectControl = this._builder.control(null, {
```

Comment:
Why can't we use `subjectName` control instead of this?

Reply:
The `km-combobox` component returns a value which is nested inside an object (`{ select: <actual data> }`), which is not representative of the payload I need to send to the API, this is why I'm handling it in a separate formControl.